velox/docs/develop/testing/fuzzer.rst

```diff
@@ -178,14 +178,35 @@ aggregate functions supported by the engine, and call
 ``AggregationFuzzerRunner::run()`` defined in `AggregationFuzzerRunner.h`_. See
 `AggregationFuzzerTest.cpp`_.
 
-.. _AggregationFuzzerRunner.h: https://github.com/facebookincubator/velox/blob/main/velox/exec/tests/AggregationFuzzer.h
+.. _AggregationFuzzerRunner.h: https://github.com/facebookincubator/velox/blob/main/velox/exec/fuzzer/AggregationFuzzer.h
 
-.. _AggregationFuzzerTest.cpp: https://github.com/facebookincubator/velox/blob/main/velox/exec/tests/AggregationFuzzerTest.cpp
+.. _AggregationFuzzerTest.cpp: https://github.com/facebookincubator/velox/blob/main/velox/functions/prestosql/fuzzer/AggregationFuzzerTest.cpp
 
 Aggregation Fuzzer allows to indicate functions whose results depend on the
 order of inputs and optionally provide custom result verifiers. The Fuzzer
 also allows to provide custom input generators for individual functions.
 
+Integration with the Window Fuzzer is similar to Aggregation Fuzzer. See
+`WindowFuzzerRunner.h`_ and `WindowFuzzerTest.cpp`_.
+
+`WindowFuzzerTest.cpp`_ allows results to be verified against Presto. To setup
+Presto as a reference DB, please follow these `instructions`_. Once the Presto
+cluster is running, the url should be passed in the argument ``--presto_url``,
```

Comment:
I think you can remove the 'once the Presto cluster is running' etc instead write it as: The following flags control the connection to the presto cluster; ``--presto_url`` which is the http server url along with its port number and ``--req_timeout_ms`` which sets the request timeout in milliseconds. The timeout is set to 1000 ms by default but can be increased it this time is insufficient for certain queries.

Reply:
Thanks, updated.